rclcpp_action/package.xml

```diff
@@ -14,6 +14,7 @@
   <author email="dthomas@openrobotics.org">Dirk Thomas</author>
   <author email="jacob@openrobotics.org">Jacob Perron</author>
 
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
Actually, we can just remove this completely.  This package already has a `buildtool_depend` on `ament_cmake_ros`, so this is redundant.

```suggestion
```

Reply:
Sounds good, removed.